### PR TITLE
to fix bugzilla 1424946

### PR DESCRIPTION
**Pull request by sdminonne**
@soltysh mind having a look?
Thanks

### CODE REVIEW

**Comment by soltysh** [comment]
[test]

File: pkg/cmd/admin/policy/subject_review.go [hunk 1]
```diff
@@ -85,12 +85,17 @@ func (o *sccSubjectReviewOptions) Complete(f *clientcmd.Factory, args []string,
 	if len(o.User) > 0 && len(o.serviceAccount) > 0 {
 		return fmt.Errorf("--user and --serviceaccount are mutually exclusive")
 	}
-	if strings.HasPrefix(o.serviceAccount, serviceaccount.ServiceAccountUsernamePrefix) {
-		_, user, err := serviceaccount.SplitUsername(o.serviceAccount)
-		if err != nil {
-			return err
+	if len(o.serviceAccount) > 0 { // check whether user supplied a list of SA
+		if len(strings.Split(o.serviceAccount, ",")) > 1 {
+			return fmt.Errorf("only one Service Account is supported.")
```
Comment (soltysh) [inline pkg/cmd/admin/policy/subject_review.go]: Just one tiny nit, not dots (`.`) at the end of error messages.

**Comment by sdminonne** [comment]
@soltysh nit fixed

**Comment by soltysh** [comment]
LGTM [merge]

**Comment by soltysh** [comment]
This suffered from https://github.com/openshift/origin/issues/13046, as well. But I'm not gonna re-kick tests.

**Pull request merged**